python/tvm/build_module.py

```diff
@@ -378,6 +378,18 @@ def lower(sch,
     for f in lower_phase0:
         stmt = f(stmt)
     # Phase 1
+    try:
+        # device_type 2 for GPU
+        # choose device 0
+        # attr type 4 for CUDA Compute Capability
+        cuda_compute_capability = _api_internal._GetDeviceAttr(2, 0, 4)
+        from tvm.contrib.nvcc import find_cuda_path, get_cuda_version
```

Comment:
seems like not necessary to put this "from..import" inside Try, and it may cause another problem that once "tvm.contrib.nvcc" changed , this module would set cuda_compute_capability into None instead of report the error.

Reply:
Yeah, we have same concerns about this part when submitting this pr. It should be better to move the cuda version and capability check to somewhere inside the TensorCore pass.